CMR/python/cmr/search/granule.py

```diff
@@ -114,9 +135,25 @@ def open_api(section='#granule-search-by-parameters'):
     """
     scom.open_api(section)
 
+def set_logging_to(level):
+    """
+    Set the logging level to the stated value
+    Parameters:
+        level: a value like logging.INFO
```

Comment:
Let's enumerate valid values here or give a link to where they are defined.

Reply:
can do

Reply:
change made for both granule and collection search code